tests/pq/test_pipeline.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+import psycopg
+from psycopg import pq
+
+pytestmark = pytest.mark.libpq(">= 14")
```

Comment:
Could you please add tests to verify that on libpq < 14 the functions raise the expected exception? Thank you

Reply:
Done.